src/RetireNet.Packages.Tool/Services/RetireServiceOptions.cs

```diff
@@ -9,5 +9,9 @@ public class RetireServiceOptions
         public string Path { get; set; }
 
         public bool AlwaysExitWithZero { get; set; }
+        public string ReportPath { get; set; }
+        public String ReportFormat { get; set; }
```

Comment:
Let's use lowercase s `string` for consistency

Reply:
Sure thing. I work in C#, JavaScript, Apex (Salesforce), and C++ pretty much daily lately.... it's amazing half of the stuff I write ends up matching the language it's written in... lol!